internal/plugins/manifests/v2/init.go

```diff
@@ -32,7 +32,7 @@ import (
 
 // Version of `opm` to download and use for building index images.
 // This version's release artifacts *must* contain a binary for multiple arches; certain releases do not.
-const opmVersion = "v1.15.1"
+const opmVersion = "v1.18.0"
```

Comment:
The latest one shows to be 1.19.1 WDYT about updating it?

Reply:
You're right. Done.